Tasks/XamariniOS/task.json

```diff
@@ -160,6 +169,7 @@
         },
         {
             "name": "provProfileUuid",
+            "aliases": [ "signingProvProfileUuid" ],
```

Comment:
In my opinion we should not abbreviate so "ProvProfile" would be "ProvisioningProfile"